### PR TITLE
Adding namespace definition for backup component

### PR DESCRIPTION
**Pull request by jrosental**

### CODE REVIEW

**Review by amfred — APPROVED** [review]

File: components/backup/base/kustomization.yaml [hunk 1]
```diff
@@ -3,3 +3,5 @@ kind: Kustomization
 namespace: openshift-adp
 resources:
   - external-secret.yaml
+  - namespace.yaml
+
```
Comment (hugares) [inline components/backup/base/kustomization.yaml:7]: nit: keep only one blank line at the end of file, there are 2 currently

File: components/backup/base/namespace.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-adp
+
```
Comment (hugares) [inline components/backup/base/namespace.yaml:5]: nit: keep only one blank line at the end of file, there are 2 currently

**Review by hugares — APPROVED** [review]
/lgtm

**Pull request merged**